gcp/service.go

```diff
@@ -47,6 +48,26 @@ func AccessApprovalService(ctx context.Context, d *plugin.QueryData) (*accessapp
 	return svc, nil
 }
 
+func APIService(ctx context.Context, d *plugin.QueryData) (*apikeys.Service, error) {
+	// have we already created and cached the service?
+	serviceCacheKey := "APIService"
```

Comment:
Is this the right cache key, if the service is is API Keys? Should it be `APIKeysService` instead?